packages/asl-service/lib/middleware/flash-middleware.js

```diff
@@ -0,0 +1,17 @@
+module.exports = function(req, res, next) {
+  if (!req.session.flash) req.session.flash = {};
+
+  // Set a flash message
+  res.flash = (key, value) => {
+    req.session.flash[key] = value;
+  };
+
+  // Get a flash message once
+  res.locals.getFlash = (key) => {
```

Comment:
res.locals are serialised, so we can't use a function here, also this isn't being called.